agent/xds/xds_protocol_helpers_test.go

```diff
@@ -177,7 +177,9 @@ func newTestServerScenarioInner(
 		nil, /*checkFetcher HTTPCheckFetcher*/
 		nil, /*cfgFetcher ConfigFetcher*/
 	)
-	s.AuthCheckFrequency = authCheckFrequency
+	if authCheckFrequency > 0 {
```

Comment:
FYI: this tweak fixed a fun logging issue where in unit tests the ACL check would happen _very fast_ and _very often_.